src/system/RoutesModule/Resources/views/Route/view.tpl

```diff
@@ -8,39 +8,45 @@
 {if count($items) > 1}
 {pageaddvar value='jquery-ui' name='javascript'}
 <script type="text/javascript">
-    jQuery(function(){
-        // Return a helper with preserved width of cells
-        var fixHelper = function(e, ui) {
-            ui.children().each(function() {
-                jQuery(this).css({width: jQuery(this).width()});
-            });
-            return ui;
-        };
+    /* <![CDATA[ */
+    ( function() {
```

Comment:
IANAJSP (I Am Not A JavaScript Programmer)

but the way I understand it , doing it your way just defines `$ = jQuery` and allows it to be used as such. I figured the way I did it just stuck with the `jQuery` var name. no?